src/Generator.cpp

```diff
@@ -880,8 +880,16 @@ int generate_filter_main_inner(int argc, char **argv, std::ostream &cerr) {
             return 1;
         }
 
-        if (LoadLibraryW(wide_lib.data()) != nullptr) {
+        if (!LoadLibraryW(wide_lib.data())) {
+            DWORD last_err = GetLastError();
+            LPVOID last_err_msg;
+            FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM |
+                              FORMAT_MESSAGE_IGNORE_INSERTS,
+                          nullptr, last_err, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
+                          reinterpret_cast<LPTSTR>(&last_err_msg), 0, nullptr);
             cerr << "Failed to load: " << lib << "\n";
+            cerr << "LoadLibraryW failed with error " << last_err << ": "
+                 << std::string(static_cast<LPCTSTR>(last_err_msg)) << "\n";
```

Comment:
This may not compile if UNICODE is defined, as LPCSTR will be a wide string and I'm not sure if std::string ctor will accept it.